src/WebJobs.Extensions.CosmosDB/Config/CosmosDBExtensionConfigProvider.cs

```diff
@@ -124,16 +124,17 @@ internal string ResolveConnectionString(string attributeConnectionString)
             return _options.ConnectionString;
         }
 
-        internal ICosmosDBService GetService(string connectionString)
+        internal ICosmosDBService GetService(string connectionString, string preferredLocations = "", bool useMultipleWriteLocations = false)
```

Comment:
nit: use `string.Empty` here.

Reply:
This cannot be done, trying to use string.Empty has a compile-time error since only compile-time values can be used.